gcc_test_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+  cat <<EOF
+  usage: $0
+  This script fetches and runs available test suites using runners
+  provided in ./*-tests-runner dirs.
+EOF
+}
+
+run_tests() {
+  local script_dir="$( cd "$(dirname "${BASH_SOURCE[0]}" )" && pwd )"
+  local tests_results_dir="$HOME/tests-results"
+  mkdir -p "$tests_results_dir"
+
+  # Fetch and run OFC tests
+  local ofc_tests_git_src="https://github.com/CodethinkLabs/ofc-tests.git"
+  local ofc_tests_dir="$HOME/ofc-tests"
+
+  if [ ! -d "$ofc_tests_dir" ]; then
+    git clone "$ofc_tests_git_src" "$ofc_tests_dir"
+  fi
```

Comment:
If you continue at this point, it means that you are going to run the build with whatever is in that repository, so you don't have control about what is going to be run.
Please, exit on else or remove the folder and recreate it.